girder/__init__.py

```diff
@@ -29,6 +36,7 @@
 import sys
 import traceback
 
+
```

Comment:
This extra newline may or may not cause pep8 failure

Reply:
It's required as far as I know.

Reply:
It's not, but it looks like it's fine by the linter.

Reply:
Oh, I was looking at the wrong line... I'll amend it.